apps/website/src/components/events/virtual-ticket/StickerSelector.tsx

```diff
@@ -44,10 +44,10 @@ export function StickerSelector({
               {typeSafeObjectKeys(stickerPack.stickers)
```

Comment:
**Do not apply directly!**

Something like this could fix this. Not the most beautiful. Also missing the import for `notEmpty` from `@/utils/helpers`. Should replace the next filter and map too (to L 54 incl.)

```suggestion
              {typeSafeObjectKeys(stickerPack.stickers)
                .map((imageId) => {
                  const stickerPackItem = stickerPack.stickers[imageId];
                  return stickerPackItem && groupId === stickerPackItem.groupId
                    ? {
                        name: stickerPackItem.name,
                        disabled: selectedStickers.includes(imageId),
                        image: mapStickerIdToPath(
                          stickerPack.stickers,
                          imageId,
                        ),
                        imageId,
                      }
                    : undefined;
                })
                .filter(notEmpty)
```